x-pack/plugins/actions/server/builtin_action_types/servicenow/service.ts

```diff
@@ -61,9 +53,30 @@ export const createExternalService = ({
     }
   };
 
+  const findIncidents = async (params?: Record<string, string>) => {
+    const query = params
```

Comment:
A little surprised axios doesn't have a way to send a query string param object argument with the request, instead of having to build one yourself.  Does this current code handle doing quoting of `&` and `=` characters, for instance?